dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/client/jsbinding/C3ChartConf.java

```diff
@@ -52,6 +52,31 @@ static C3ChartConf create(C3ChartSize size,
         return instance;
     }
     
+    @JsOverlay
+    static C3ChartConf create(C3ChartSize size, 
+                              C3ChartData data, 
+                              C3AxisInfo axis,
+                              C3Grid grid,
+                              C3Transition transition,
+                              C3Point point,
+                              C3Padding padding,
+                              C3Legend legend,
+                              C3Color color,
+                              C3Tooltip tooltip) {
+        C3ChartConf instance = new C3ChartConf();
```

Comment:
Why we don't set all those properties via constructor?

Reply:
The issue is that C3Meter and other C3* classes from jsbinding package are just to create literal javascript objects, which means that adding a constructor will fail since we don't have a defined Javascript object in C3. C3Gauge for example, translates to:
~~~
{ "min": 10, "max": 10}
~~~

No C3Gauge object is available!